platform-mbean/src/main/java/org/jboss/as/platform/mbean/RuntimeMXBeanAttributeHandler.java

```diff
@@ -116,7 +116,11 @@ static void storeResult(final String name, final ModelNode store) {
         } else if (PlatformMBeanConstants.BOOT_CLASS_PATH_SUPPORTED.equals(name)) {
             store.set(ManagementFactory.getRuntimeMXBean().isBootClassPathSupported());
         } else if (PlatformMBeanConstants.BOOT_CLASS_PATH.equals(name)) {
-            store.set(ManagementFactory.getRuntimeMXBean().getBootClassPath());
+            if (!ManagementFactory.getRuntimeMXBean().isBootClassPathSupported()) {
+                store.set(""); // JDK9 and above does not have boot class path
+            } else {
+                store.set(ManagementFactory.getRuntimeMXBean().getBootClassPath());
+            }
```

Comment:
AFAIR there was one test failing because of this. I'll take a look if we could fix it somehow in that test instead.